docs/testing-operators.md

```diff
@@ -28,6 +28,23 @@ The scorecard assigns points to each passing component of the scorecard. The tot
 
 **Note**: no explicit number of points or percentage is necessary to achieve before merging _yet_. These are suggestions to improve your operator.
 
+##### Running scorecard locally
+
+**Note**: This section is under development, and may not work for all platforms.
+
+It's possible to run the Operator SDK scorecard locally for operator validation. If you have a minikube cluster running with OLM installed, as well as other prerequisites outlined in `scripts/ci/install-deps`, you can run the following:
```

Comment:
It might be a good idea to highlight how to get a minikube cluster with OLM. Afaik the `Makefile` has a `deploy-local` command for that purpose.

Reply:
We currently install OLM using the manifests in the 0.8.1 release, which can be set up by running `./scripts/ci/install-olm`. I can add that as well